assets/css/materialize.css

```diff
@@ -10741,3 +10741,26 @@ div.slack {
     font-size: 8px;
   }
 }
+
+/*Community Meetings Table */
+#table101 {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+#table101 td {
+  border: 1px solid #dcdcdc;
+  text-align: left;
+  padding: 8px;
+  column-gap: 80px;
+}
+
+#table101 th {
+  background-color: #20BF9F;
```

Comment:
Use "#477E96;"

Use bold font weight for column headings.

Use white for font color.

Set border-radius to 0px.